src/sagemaker_xgboost_container/algorithm_mode/hyperparameter_validation.py

```diff
@@ -202,5 +202,8 @@ def eval_metric_dep_validator(value, dependencies):
                                              required=False),
         hpv.IntegerHyperparameter(name="seed", range=hpv.Interval(min_open=-2**31, max_open=2**31-1),
                                   required=False),
+
+        # For internal use only!
+        hpv.ContinuousHyperparameter(name="_subsample_ratio_on_read", range=hpv.Interval(min_open=0, max_open=1), required=False),
```

Comment:
This line is too long. Can you break it up, one arg per line?

Reply:
I broke it up. Overall the whole code base needs to be linted though. Looks like no one has run `flake8` or a similar tool yet.